docs/resources/cnad_advanced_black_white_list.md

```diff
@@ -0,0 +1,52 @@
+---
+subcategory: "Cloud Native Anti-DDoS Advanced"
+---
+
+# huaweicloud_cnad_advanced_black_white_list
+
+Manages a CNAD advanced policy black and white list resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "policy_id" {}
+variable "black_ip_list" {
+  type = list(string)
+}
+variable "white_ip_list" {
+  type = list(string)
+}
+
+resource "huaweicloud_cnad_advanced_black_white_list" "test" {
+  policy_id     = var.policy_id
+  black_ip_list = var.black_ip_list
+  white_ip_list = var.white_ip_list
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `policy_id` - (Required, String, ForceNew) Specifies the policy id.
+
+  Changing this parameter will create a new resource.
+
+* `black_ip_list` - (Optional, List) Specifies the black ip list. The minimum length is 7. The maximum length is 128.
```

Comment:
What's the meaning: The minimum length is 7. There is only two ip in the acc test.

Reply:
I have changed the description, The length indicate the IP address or IP address range.